src/browser/modules/Sidebar/Sidebar.tsx

```diff
@@ -176,7 +176,7 @@ const Sidebar = ({
 
   return (
     <TabNavigation
-      openDrawer={openDrawer}
+      selectedDrawerName={openDrawer}
```

Comment:
It'd be nice to rename openDrawer to selectedDrawerName in the rest of the code as well 🤔 something we could do in a future PR